src/test/resources/imp2.kore

```diff
@@ -554,11 +552,11 @@ module IMP
           HOLE:AExp{},
           divAExp{}(HOLE:AExp{}, Y:AExp{})),
         X:AExp{}))
-  [strict{}(\dv{Strict{}}("divAExp"), firstArgument{}())]
+  [ strictAxiom{}("divAExp,1") ]

```

Comment:
I think that you should use divaexp as as symbol here and, even if you're not doing that, 1 should be in a different string.

Reply:
ok...